backend/de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/v2/invoice/InvoicesRestController.java

```diff
@@ -312,8 +317,28 @@ public ResponseEntity<JsonCreateInvoiceResponse> createInvoice(@NonNull @Request
 
 			return ResponseEntity.unprocessableEntity()
 					.body(JsonCreateInvoiceResponse.builder()
-								  .errors(ImmutableList.of(JsonErrors.ofThrowable(ex, adLanguage)))
-								  .build());
+							.errors(ImmutableList.of(JsonErrors.ofThrowable(ex, adLanguage)))
+							.build());
+		}
+	}
+
+	@PostMapping("/docReview")
+	public ResponseEntity<JsonCreateInvoiceReviewResponse> createReview(@NonNull @RequestBody final JsonInvoiceReviewUpsertItem jsonInvoiceReviewUpsertItem)
+	{
+		try
+		{
+			final Optional<JsonCreateInvoiceReviewResponse> response = jsonInvoiceReviewService.upsert(jsonInvoiceReviewUpsertItem);
+
+			return response.map(ResponseEntity::ok).orElseGet(() -> ResponseEntity.badRequest().build());
+		}
+		catch (final Exception ex)
+		{
+			logger.error(ex.getMessage(), ex);
```

Comment:
minor thing: i think using the exception message as message for logger is not best practice and redundant because the logger will already print that.
u could replace it with something like
```java
logger.error("Failed to create the review", ex);
```

Reply:
Was inspired by code above. Fixed here and above.